src/App.js

```diff
@@ -1,18 +1,38 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
+import axios from 'axios';
 import './App.css';
 
 class App extends Component {
+  state = {
+    gifs: []
+  }
+
+  componentDidMount() {
```

Comment:
good first start, I recommend that you start migrating to redux and add the api keys to a `.env` file.